misc/python/materialize/cli/mkrelease.py

```diff
@@ -199,20 +200,20 @@ def dashboard_links(start_time: str) -> None:
     template = (
         "https://grafana.i.mtrlz.dev/d/materialize-overview/materialize-overview-load-tests?"
         + "orgId=1&from={time_from}&to={time_to}&var-test={test}&var-purpose={purpose}"
-        + "&var-env=dev"  # &var-workflow=cloud-load-test
+        + "&var-env={env}"  # &var-workflow=cloud-load-test
```

Comment:
Could you delete the leftover comment here? it was meant to be removed  in the commit that initially added the `--env` flag

Reply:
```suggestion
        + "&var-env={env}"
```